src/activities/assignments/CategoriesEntity.js

```diff
@@ -33,16 +33,28 @@ export class CategoriesEntity extends Entity {
 	}
 
 	/**
-	 * @returns {Boolean} Whether or not a user can edit categories
+	 * @returns {Boolean} Whether or not a user can add categories
 	*/
-	canEditCategories() {
+	canAddCategories() {
 		if (!this._entity) {
 			return;
 		}
 
 		return this._entity.hasActionByName(Actions.assignments.add);
 	}
 
+	/**
+	 * @returns {Boolean} Whether or not a user can edit categories
+	*/
+	canEditCategories() {
+		const categories = this.getCategories();
+		if (!categories || !categories.length) {
+			return;
+		}
+
+		return categories[0].hasActionByName(Actions.assignments.categories.select);
```

Comment:
Is it possible that the first category will be selected, in which case we would be looking for the `deselect` action?

Reply:
That's a very good point. Changing it to check for both.